kerl

```diff
@@ -2099,6 +2099,39 @@ get_active_path() {
     fi
 }
 
+fix_otp_installations() {
+    if [ -f "$KERL_BASE_DIR"/otp_installations ]; then
+        inexisting_names=""
+        while read -r l; do
+            names=$(echo "$l" | cut -d' ' -f1)
+            path=$(echo "$l" | cut -d' ' -f2)
```

Comment:
I actually wanted to have a function that would centralise this manipulation (for getting _name_ and _path_), but then we have a `\grep -m1 -E ...` built on it that'd have to be changed to a `for` loop too, so I quit that idea.

I don't think the format of these files will change any time soon, for the sheer fact that it'd break for all consumers out there, so this should still be Ok, even if a bit more verbose.

Reply:
Resolve'ing as preparing to merge.